test/docfx.Tests/JsonConverterTest.cs

```diff
@@ -49,6 +50,51 @@ public void TestJObjectDictionaryToObjectDictionaryConverterSerializeAndDeserial
         Assert.Equal(jsonString, JsonConvert.SerializeObject(buildOptions, settings));
     }
 
+    [Fact]
+    [Trait("Related", "docfx")]
+    public void TestMarkdownEnginePropertiesSerializeAndDeserialize()
+    {
+        string jsonString = """
+                            {
+                              "markdownEngineProperties": {
+                                "enableSourceInfo": false,
+                                "alerts": {
+                                  "TODO": "alert alert-secondary",
+                                  "REVIEW": "alert alert-primary"
+                                },
+                                "plantUml": {
+                                  "outputFormat": "svg",
+                                  "remoteUrl": "https://www.plantuml.com/plantuml"
+                                }
+                              },
+                              "disableGitFeatures": true
+                            }
+                            """;
```

Comment:
It needs to add `.ReplaceLineEndings();` to compare platform invariant string assertion.
(Because current `.cs` source code use `CRLF` line separator)